phpunit.xml.dist

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         bootstrap="./vendor/autoload.php"
+         colors="true"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd">
+
+  <coverage>
+    <include>
+      <directory>src</directory>
+    </include>
+  </coverage>
+
+  <testsuites>
+    <testsuite name="MisdPhoneNumberBundle Test Suite">
+      <directory>tests</directory>
+    </testsuite>
+  </testsuites>
 
-<phpunit bootstrap="./vendor/autoload.php" colors="true">
```

Comment:
Can we keep the indentation? Thanks

Reply:
I am not sure which  indentation you exatcly mean. I changed it, can you have a look if you want it like that, or please be more precisely.